### DISCUSSION TITLE
Deploy Next JS to Firebase

### DISCUSSION THREAD

**Discussion by tranquocviet226** [community/community#38119] [Other Feature Feedback, Questions, & Ideas]
I got an error.

Anybody help me. Thank you.

![image](https://user-images.githubusercontent.com/57559641/199629074-91c6fa04-d23f-46c1-8c1f-510e62fc4944.png)

**Comment by jfullstackdev** [comment]
can you give further context on that ? 

are you deploying on production server ? what are the tools you are using ?
